docs/events/eboard-meetings.rst

```diff
@@ -0,0 +1,49 @@
+###############
+Eboard Meetings
+###############
+
+This page documents RITlug executive board (eboard) meetings.
+Topics include room booking, preparation, and running of the meeting.
+
+
+*****************
+Meeting checklist
+*****************
+
+Pre-meeting
+===========
+
+The President will triage the `RITlug operations board <https://github.com/orgs/RITlug/projects/1>`__, looking for tasks that require attention.
+The label **review at meeting** should be assigned to tasks that need explicit review by eboard.
+
+Each topic to be discussed at an eboard meeting must have a corresponding issue.
+
+Meeting
+=======
+
+#. Designate a notetaker, typically the `secretary <../roles/secretary>`__, to record discussion points and action items for each task.
+   The notetaker should record their notes in the comments section of each task.
+   At the end of each eboard meeting every task should have a clear set of action items to be worked on.
+
+#. A member will lead the meeting by reviewing the tasks assigned the **review at meeting** `label <https://github.com/orgs/RITlug/projects/1?card_filter_query=label%3A%22review+at+meeting%22>`__.
+
+If new work is discussed during the meeting then a task should be created to track the work within the `RITlug operations board <https://github.com/orgs/RITlug/projects/1>`__.
+
+*************
+Date and time
+*************
+
+At the start of the semester, a day of week and time should be decided upon by the eboard members during which they are all free to meet on campus.
+Eboard members who are on co-op or otherwise unavailable for in-person meetings during the semester have the option to be videoconferenced into the meeting.
+
+Holding an eboard meeting several days ahead of the general meeting provides eboard the time to react to last-minute issues, such as speaker cancellations or content review.
+
+**************
+Booking a room
+**************
+
+A room equipped with a projector or shared display is useful for displaying the RITlug operations board.
+Booking a room for the semester is recommended.
+
+If this is not an option then a library study room can be reserved on behhalf of the club using `EMS <event-management-system.html>`__
```

Comment:
The *only* nitpick I have, is that there is no period after EMS before leading into the next sentence. It's not a super deal breaker, so if others find this fine, or if you are able to get in a small commit to fix this, then it's all good ^.^

I'm going to approve these changes, but keep this in mind.

Reply:
I have nothing else to add to my review for this pull request. @icflournoy, feel free to get this, rebase, and merge on push. :+1:

Reply:
Oops! I just noticed this doesn't use the Sphinx cross-document syntax. The correct syntax here is:

```rst
:doc:`EMS <event-management-system>`
```